README.md

```diff
@@ -1,217 +1,98 @@
-its `tail`. The second `app-route` consumes the `subroute`, and when it
-matches, it produces an object called `subrouteData` from its `data`.
-
-So, when `route.path` is `'/about'`, the `routeData` object will look like
-this: `{ page: 'about' }`
-
-And `subrouteData` will be null. However, if `route.path` changes to
-`'/article/123'`, the `routeData` object will look like this:
-`{ page: 'article' }`
-
-And the `subrouteData` will look like this: `{ id: '123' }`
-
-`app-route` is responsive to bi-directional changes to the `data` objects
-they produce. So, if `routeData.page` changed from `'article'` to `'about'`,
-the `app-route` will update `route.path`. This in-turn will update the
-`app-location`, and cause the global location bar to change its value.
-
-
-
-## &lt;app-location&gt;
-
-`app-location` is an element that provides synchronization between the
-browser location bar and the state of an app. When created, `app-location`
-elements will automatically watch the global location for changes. As changes
-occur, `app-location` produces and updates an object called `route`. This
-`route` object is suitable for passing into a `app-route`, and other similar
-elements.
+See: [Documentation](https://www.webcomponents.org/element/@polymer/app-route),
+  [Large Demo](https://www.webcomponents.org/element/@polymer/app-route/demo/demo/index.html),
+  [Simple Demo](https://www.webcomponents.org/element/@polymer/app-route/demo/demo/simple-demo.html),
+  [Data Loading Demo](https://www.webcomponents.org/element/@polymer/app-route/demo/demo/data-loading-demo.html).
 
-An example of the public API of a route object that describes the URL
-`https://elements.polymer-project.org/elements/app-location`:
+## Usage
 
-```css
-{
-  prefix: '',
-  path: '/elements/app-location'
-}
+### Installation
 ```
-
-Example Usage:
-
-```html
-<app-location route="{{route}}"></app-location>
-<app-route route="{{route}}" pattern="/:page" data="{{data}}"></app-route>
+npm install --save @polymer/app-route
 ```
 
-As you can see above, the `app-location` element produces a `route` and that
-property is then bound into the `app-route` element. The bindings are two-
-directional, so when changes to the `route` object occur within `app-route`,
-they automatically reflect back to the global location.
-
-### Hashes vs Paths
-
-By default `app-location` routes uses the pathname portion of the URL. This has
-broad browser support but it does require cooperation of the backend server. An
-`app-location` can be configured to use the hash part of a URL instead using
-the `use-hash-as-path` attribute, like so:
-
+### In an HTML file
 ```html
-<app-location route="{{route}}" use-hash-as-path></app-location>
+<html>
+  <head>
+    <script type="module">
+      import '@polymer/app-route/app-location.js';
+      import '@polymer/app-route/app-route.js';
+    </script>
+  </head>
+  <body>
+    <app-location route="{{route}}"></app-location>
+    <app-route
+        route="{{route}}"
```

Comment:
This doesn't work in an html file like this...

Reply:
ah

Reply:
fixed